cmd/snap-confine/seccomp-support.c

```diff
@@ -35,6 +36,21 @@
 #include "../libsnap-confine-private/string-utils.h"
 #include "../libsnap-confine-private/utils.h"
 
+#ifndef SECCOMP_FILTER_FLAG_LOG
+#define SECCOMP_FILTER_FLAG_LOG 2
+#endif
+
+#ifndef seccomp
```

Comment:
Is seccomp defined in the pre-processor usually?

Reply:
No, the `seccomp()` syscall may not have a glibc wrapper:

 https://lwn.net/Articles/655028/

It is important that we wrap it in that situation.